.travis.yml

```diff
@@ -2,12 +2,27 @@ language: c
 sudo: required
 env:
   - TARGET=local
-  - TARGET=docker
+  - TARGET=docker OS=ubuntu-16.04
+  - TARGET=docker OS=fedora-25
 services:
   - docker
 before_install:
-  - if [ ${TARGET}="local" ];  then sudo apt-get update && sudo apt-get install libsqlite3-dev; fi
-  - if [ ${TARGET}="docker" ]; then sudo docker build -t "cangjians/libcangjie:latest" . ; fi
+  - | # before_install: local
+    if [ "${TARGET}" = "local"  ]; then
```

Comment:
Since that doesn't work in the (ancient) Travis setup, should we just remove it completely?